src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -775,37 +775,3 @@ services:
   PrestaShopBundle\Form\Admin\Type\TaxInclusionChoiceType:
   PrestaShopBundle\Form\Admin\Type\ShippingInclusionChoiceType:
   PrestaShopBundle\Form\Admin\Type\ProductSearchType:
-
```

Comment:
Attention, you're removing service that has been deprecated **in** 9.0

Reply:
Sorry. That lacks some comments.